server.js

```diff
@@ -0,0 +1,30 @@
+const express = require('express');
+const home = require('./routes/home');
+const register = require('./routes/register');
+const login = require('./routes/login');
+const mongoose = require('mongoose');
+const path = require('path');
+
+// connection to database 
+mongoose.connect("mongodb://localhost/users_storage", (err, data) => {
+    if (err) {
+        console.log(err);
+        return
+    }
+    else console.log("Connection to database sucessfull");
```

Comment:
Code style: suggestion to wrap the else part in brackets as in the if part to have a more consistent code style.